src/ratingController.js

```diff
@@ -1,65 +1,69 @@
-const fetch = require('node-fetch')
+import fetch from "node-fetch";
 
-module.exports = {
-    getAuthorities: async function (req, res) {
-        const requestParams = {
-            headers: {
-                Accept: 'application/json',
-                'x-api-version': '2',
-            },
-        };
+async function getAuthorities(req, res) {
+  const requestParams = {
+    headers: {
+      Accept: "application/json",
+      "x-api-version": "2",
+    },
+  };
 
-        let authoritiesResponse = null;
+  let authoritiesResponse = null;
 
-        try {
-            authoritiesResponse = await fetch('http://api.ratings.food.gov.uk/Authorities', requestParams);
-        } catch (err) {
-            console.log(err);
-            return res.status(500).json({ error: 'Unable to access FSA API' });
-        }
+  try {
+    authoritiesResponse = await fetch(
+      "http://api.ratings.food.gov.uk/Authorities",
+      requestParams,
+    );
+  } catch (err) {
+    console.log(err);
```

Comment:
Maybe some useless comment from my side, but:

`err` might be an object when logged in the server side you might get something like: [object, Object].

We could do:

console.error(JSON.stringify(err));